app/Http/Controllers/SupplierController.php

```diff
@@ -33,4 +34,37 @@ public function index()
         $suppliers = $this->supplier->orderBy('id', 'ASC')->paginate(10);
```

Comment:
should not use `hardcode` on paginate, you can define `limit Pagination` on Model `Supplier`.